### ISSUE TITLE
Remove publishConfig block from package.json

### ISSUE THREAD

**Issue by yrtimiD** [mridgway/hoist-non-react-statics#92]
Having publishConfig with hardcoded global registry makes impossible to upload this package to custom local offline mirrors (usually used by companies):
```
"publishConfig":` {
    "registry": "https://registry.npmjs.org/"
  }
```
Removing it should not affect regular version release since registry.npmjs.org is default in npm.

**Comment by mridgway** [comment]
I added this to the config because I am also at a company that has an internal registry and I routinely forget to specify the correct registry when publishing this library to the open source registry.

If you want to publish it to your own registry you should be able to run `npm publish --registry https://registry.mycompany.com`. I didn't think the `publishConfig` overrode the cli parameter, but I could be mistaken.

**Comment by yrtimiD** [comment]
Right, specifying explicit registry should work, but makes life harder. We usually use only .npmrc files, and never mention registry explicitly.

Anyway, it's your call, feel free to close the issue :)

**Comment by hazolsky** [comment]
I recently ran into the same issue and specifying `--registry` didn't help either.

Maybe it'd be more convenient for both you and community if registry URL was stored in per-project config file (`/path/to/my/project/.npmrc`)?
